### ISSUE TITLE
wk2 sept_epic Bug: Sankey Diagram - Hover data values on leftside not showing thru Teams screen share (displayed locally, but not not shared)

### ISSUE THREAD

**Issue by dlkibbe** [lzim/teampsd#2716]
**1. Add description:** During Facilitate Workgroup Session on 8/23/2022, @swapmush was sharing his screen using Edge and showing the Sankey diagram. The pop-up boxes that contain veteran patient numbers when hovering over the bars on the Sankey diagram were not appearing to the MTL facilitators on Teams.
I logged into Google Chrome and can see the pop-up boxes when I hover over the vertical bars in the Sankey.

<< Paste screenshots here>>
![image](https://user-images.githubusercontent.com/31182258/186214351-b7a33e5e-6089-49fc-b13f-4aaff53317cb.png)

1. When was the bug discovered? 8/23/2022, 12:30 pm, MTL Facilitate Workgroup Session
2. Is this an individual or shared world? @swapmush individual world
3. What is the name of the experiment? 
 
**2. Click on the right:**  
@swapmush and @jamesmrollins tagged

**3. If you anticipate that your **issue card** or **bug card** will have **dependencies** these can be added once the issue has been submitted.   
- this may be a feature that is not supported by Teams vs. a bug in the SIM UI

ix. **Add dependencies** - this may be a feature that is not supported by Teams vs. a bug in the SIM UI

**Comment by jamesmrollins** [comment]
I visited with @swapmush and he shared his screen with me and demonstrated how the popup on the Sankey chart was not visible.  To test again, I set up a similar instance using the East Coast Virtual desktop and brought myself into a meeting.  I was able to see all popups associated with the Sankey chart.  @swapmush and I should meet again and see if this phenomenon persists.  It may be a communication speed issue related with the ability of Teams to refresh shared screens.
